src/pages/Campaign/LeaderboardLayout.tsx

```diff
@@ -113,7 +113,7 @@ export default function LeaderboardLayout({
         </LeaderboardTableBodyItem>
         {showRewardsColumn && (
           <LeaderboardTableBodyItem align="right" isThisRankingEligible={isThisRankingEligible}>
-            {data.rewardAmount.toSignificant(DEFAULT_SIGNIFICANT)} {data.token.symbol}
+            {data.rewardAmount.toSignificant(DEFAULT_SIGNIFICANT)} {data?.token?.symbol}
```

Comment:
should be `{data?.token?.symbol || ''}`

Reply:
It doesn't render `undefined` to the screen but it is better for the screen reader.

Reply:
Im not sure in screen reader case. How does it work in this case? @nguyenhuudungz

Reply:
![image](https://user-images.githubusercontent.com/34548156/181215895-cc99274a-037c-4cfc-a809-9da2c5b2ad20.png)